src/network/networkd-route.c

```diff
@@ -863,3 +868,41 @@ int config_parse_route_table(const char *unit,
 
         return 0;
 }
+
+int config_parse_route_device(const char *unit,
+                           const char *filename,
+                           unsigned line,
+                           const char *section,
+                           unsigned section_line,
+                           const char *lvalue,
+                           int ltype,
+                           const char *rvalue,
+                           void *data,
+                           void *userdata) {
+        _cleanup_route_free_ Route *n = NULL;
```

Comment:
Strange indentation: it's best to move the first argument into a line of its own and indent all parameters the same.